test/e2e/layers/layerLabel.test.js

```diff
@@ -0,0 +1,34 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe('Layer Label Tests', () => {
+  let page;
+  let context;
+  test.beforeAll(async function () {
+    context = await chromium.launchPersistentContext('');
+    page =
+      context.pages().find((page) => page.url() === 'about:blank') ||
+      (await context.newPage());
+    await page.goto('layerLabel.html');
+  });
+
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test('Name of unnamed layer is Layer', async () => {
+    const label = await page.locator('body > mapml-viewer > layer-');
```

Comment:
In terms of meaningful variables, label is actually the layer.  I think you could do page.locator(...).evaluate() to return the actual label string.  Of course you would have to do that after the waitForTimeout, because locators work fast but evaluates actually access the attribute in the page.  Since you had already done the await, you could do the expect without the await inside.